platform/solr/solr-factory/pom.xml

```diff
@@ -50,6 +106,16 @@
             <groupId>com.google.guava</groupId>
             <artifactId>guava</artifactId>
         </dependency>
+        <dependency>
+            <groupId>net.jodah</groupId>
+            <artifactId>failsafe</artifactId>
+            <version>0.9.3</version>
```

Comment:
If they were in the same app or if they were not embedded then I would agree but since neither case is true yet I think it can stay this way for now.